app/components/inputs/search-box.jsx

```
@@ -0,0 +1,25 @@
+/**
+ * SearchBox component renders a search input box with a search button.
+ * 
+ * @param {Object} props - The properties object.
+ * @param {string} [props.className] - Optional additional class names for the search box.
+ * @param {string} [props.placeholder] - Optional placeholder text for the input.
+ * @param {string} [props.text] - Optional initial text value for the input.
+ * @returns {JSX.Element} The rendered search box component.
+ */
+export default function SearchBox(props) {
+    return(
+        <div className={`sm:grow sm:self-stretch sm:justify-stretch flex search-box rounded-full h-fit sm:h-14 sm:max-w-[720px] bg-[--md-sys-color-surface-container] text-[--md-sys-color-on-surface-container] ${props.className}`} role='search'>
```

Biome

[error] The elements with the following roles can be changed to the following elements:
<input type="search">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)